docs/configuration/authentication/file.md

```diff
@@ -27,50 +27,85 @@ The format of the file is as follows.
             groups:
                 - admins
                 - dev
 
         harry:
-            password: "$6$rounds=500000$jgiCMRyGXzoqpxS3$w2pJeZnnH8bwW3zzvoMWtTRfQYsHbWbD/hquuQ5vUeIyl9gdwBIt6RWk2S6afBA0DPakbeWgD/4SZPiS0hYtU/"
+            password: "$argon2id$v=19$m=65536,t=3,p=2$BpLnfgDsc2WD8F2q$o/vzA4myCqZZ36bUGsDY//8mKUYNZZaR0t4MFFSs+iM"
             email: harry.potter@authelia.com
             groups: []
 
         bob:
-            password: "$6$rounds=500000$jgiCMRyGXzoqpxS3$w2pJeZnnH8bwW3zzvoMWtTRfQYsHbWbD/hquuQ5vUeIyl9gdwBIt6RWk2S6afBA0DPakbeWgD/4SZPiS0hYtU/"
+            password: "$argon2id$v=19$m=65536,t=3,p=2$BpLnfgDsc2WD8F2q$o/vzA4myCqZZ36bUGsDY//8mKUYNZZaR0t4MFFSs+iM"
             email: bob.dylan@authelia.com
             groups:
                 - dev
 
         james:
-            password: "$6$rounds=500000$jgiCMRyGXzoqpxS3$w2pJeZnnH8bwW3zzvoMWtTRfQYsHbWbD/hquuQ5vUeIyl9gdwBIt6RWk2S6afBA0DPakbeWgD/4SZPiS0hYtU/"
+            password: "$argon2id$v=19$m=65536,t=3,p=2$BpLnfgDsc2WD8F2q$o/vzA4myCqZZ36bUGsDY//8mKUYNZZaR0t4MFFSs+iM"
             email: james.dean@authelia.com
 
 This file should be set with read/write permissions as it could be updated by users
 resetting their passwords.
 
 ## Passwords
 
-The file contains hash of passwords instead of plain text passwords for security reasons.
+The file contains hashed passwords instead of plain text passwords for security reasons.
 
-You can use authelia binary or docker image to generate the hash of any password.
+You can use Authelia binary or docker image to generate the hash of any password.
 
 For instance, with the docker image, just run
 
     $ docker run authelia/authelia:latest authelia hash-password yourpassword
-    $6$rounds=50000$BpLnfgDsc2WD8F2q$be7OyobnQ8K09dyDiGjY.cULh4yDePMh6CUMpLwF4WHTJmLcPE2ijM2ZsqZL.hVAANojEfDu3sU9u9uD7AeBJ/
+    $argon2id$v=19$m=65536,t=3,p=2$BpLnfgDsc2WD8F2q$RNAy4ppk5taziCtvH48b6PadEz7r88vZV5n7WmU7yGk
 
 
 ## Password Hash Function
 
-The only supported hash function is salted sha512 determined by the prefix `$6$` as described
-in this [wiki](https://en.wikipedia.org/wiki/Crypt_(C)) page. 
-
-Although not the best hash function, Salted SHA512 is a decent algorithm given the number of rounds is big
-enough. It's not the best because the difficulty to crack the hash does not on the performance of the machine.
-The best algorithm, [Argon2](https://en.wikipedia.org/wiki/Argon2) does though. It won the
-[Password Hashing Competition](https://en.wikipedia.org/wiki/Password_Hashing_Competition) in 2015 and is now
-considered the best hashing function. There is an open [issue](https://github.com/authelia/authelia/issues/577)
-to add support for this hashing function.
+The only supported hash functions are salted Argon2id (default, version 19 only), and salted SHA512 for backwards compatibility; determined
+by the prefix `$argon2id$` and `$6$` respectively, as described in this [wiki](https://en.wikipedia.org/wiki/Crypt_(C)) page. 
```

Comment:
```suggestion
These are determined by the prefix `$argon2id$` and `$6$` respectively, as described in this [wiki](https://en.wikipedia.org/wiki/Crypt_(C)) page. 
```

Reply:
Done, also the (C) had to be (C\) due to markdown.